Adds a description to each of the libraries that Plaid is using

amilcar-andrade opened the pull request:
Adds a description to each of the libraries that Plaid is using.

Also, changes public modifier to package modifier for some classes and
variables

@nickbutcher please review and merge if you think is a good improvement.

![screenshot_20161010-204225](https://cloud.githubusercontent.com/assets/802308/19255285/550ad2e0-8f2a-11e6-8a27-8a2264c08f25.png)

amilcar-andrade commented on app/src/main/res/layout/library.xml:
```diff
@@ -50,12 +50,22 @@
+        android:layout_toEndOf="@id/library_image"
+        android:layout_alignParentEnd="true"
+        android:maxLines="2"
+        style="@style/TextAppearance.About" />
```
I reused this style not sure if this is the best one.

nickbutcher commented:
Thanks for the contribution! It mostly looks good. Sorry to be picky but could you move the text styling into a style? Thanks!

amilcar-andrade commented:
@nickbutcher sure, how should I name the style? Thanks!

amilcar-andrade commented on app/src/main/res/values/styles.xml:
```diff
@@ -370,6 +370,16 @@
         <item name="android:letterSpacing">0.01</item>
     </style>
 
+    <style name="Widget.Plaid.SecondaryText">
```
@nickbutcher please review

nickbutcher commented on app/src/main/res/values/styles.xml:
```diff
@@ -370,6 +370,16 @@
 
+    <style name="Widget.Plaid.SecondaryText">
+        <item name="android:textAppearance">@style/TextAppearance.SecondaryText</item>
+        <item name="android:maxLines">2</item>
```
Can you move the `maxLines` attribute back into the layout as this is layout info, not styling? Thanks!

nickbutcher replied:
No let's leave `CollapsingTitle` and just address the libraries in this change.

I should probably make the `CollapsingTitle` style a default style for `CollapsingTitleLayout` rather than having to provide it in the layout def…

amilcar-andrade commented:
@nickbutcher I addressed your changes. Also, I added `android:letterSpacing">0.01` to be consistent with the letter spacing of `TextAppearance.Subhead`.

amilcar-andrade commented on app/src/main/res/values/styles.xml:
```diff
@@ -370,6 +370,16 @@
+
+    <style name="TextAppearance.SecondaryText" parent="@android:style/TextAppearance.Material.Body1">
+        <item name="android:textColor">@color/text_tertiary_dark</item>
+        <item name="android:textSize">14sp</item>
```
Yeah, sorry! I noticed that last nite that is why I selected `Material.Body1` because the dimen is the following one` <dimen name="text_size_body_1_material">14sp</dimen>` .

My bad sorry for not removing it. I just did

nickbutcher commented:
Thanks for the changes!

amilcar-andrade commented:
@nickbutcher np
